test/mock/test-modules.js

```diff
@@ -117,3 +117,35 @@ angular.module('ncy-sample-conf', ['ncy-sample', 'ngMock']).config(function($url
     $httpBackend.when('GET', 'views/room_detail.html').respond('dummy room_detail view');
     $httpBackend.when('GET', 'views/room_form.html').respond('dummy room_form view');
 });
+
+/**
+ * Module with abstract state with footer view
+ */
+angular.module('ncy-abstract-interpolation-conf', []).config(function($stateProvider) {
+    $stateProvider
+        .state('A', {
+            abstract: true,
+            url: '/a',
+            views: {
+                'a@': {
+                    template: '<div>View A</div>',
+                    controller: 'ACtrl'
+                }
+            }
+        })
+        .state('A.B', {
+            url: '/b',
+            views: {
+                'b@': {
+                    template: '<div>View B</div>',
+                    controller: 'BCtrl'
+                }
+            },
+            ncyBreadcrumb: {
+                label: 'State {{tripleB}}'
+            }
+        });
```

Comment:
I think we could enrich this configuration with real multiple views to demonstrate it fixes #62 too, but I can add it later.